doc-content/enterprise-only/getting-started/business-process-proc.adoc

```diff
@@ -26,77 +26,95 @@ The *Package* specifies the location inside the existing project where the asset
 . Scroll down and expand *Process Data* and click image:getting-started/btn_plus.png[] in the *Process Variables* section.
 . Enter the following values:
 +
-* *Name*: `application`.
+* *Name*: `application`
 * *Data Type*: `Application [com.myspace.mortgage_app]`
 
 == Creating outgoing connections and exclusive gateways
-Create an outgoing connection to an exclusive gateway. Exclusive gateways are used to make decisions and react to events based on the available data.
+This section shows you how to create outgoing connections, exclusive gateways, and business rule tasks. Exclusive gateways are used to make decisions and react to events based on the available data.
 
 .Procedure
-. Click on the start event and create an outgoing connection from the start event to an exclusive gateway.
+
+. Create an outgoing connection from the start event to an exclusive gateway:
+.. Click the start event and click the *Create Parallel* icon.
++
+image::getting-started/create-parallel.png[]
+.. Hover over the image:getting-started/config-icon.png[] icon of the parallel and click the *Convert into Exclusive* icon.
 +
 image::getting-started/exclusive-gateway.png[]
 
-. Create an outgoing connection from the exclusive gateway to a task.
+. Create an outgoing connection from the exclusive gateway to a business rule task:
+.. Click the exclusive gateway and click the *Create Task* icon.
 +
 image::getting-started/create-task.png[]
-
-. Convert the new task to a business rule task.
+.. Hover over the image:getting-started/config-icon.png[] icon of the task and click the *Convert into Business Rule* icon.
 +
 image::getting-started/bus-rule-task.png[]
 
-. Click on the business rules task and enter `Validation` in the *Name* field of the *Diagram properties* panel.
-. Expand *Implementation/Execution*, select *validation* in the *Rule Flow Group* field.
-. Define the following Java expression in the *On Exit Action* field :
+. Configure the business rule task:
+
+.. Click the business rules task.
+.. If the *Diagram properties* panel is not open, in the upper-right corner click the *Diagram properties* image:getting-started/diagram_properties.png[] icon.
+.. In the *Diagram properties* panel, enter `Validation` in the *Name* field.
+.. Expand *Implementation/Execution* and from the *Rule Flow Group* list select *validation*.
+.. In the *On Exit Action* field, enter the following Java expression:
```

Comment:
Is it important to know that this is a Java expression?

Reply:
As this is part of a getting started guide, I think it doesn't hurt.